Makefile

```diff
@@ -83,7 +83,7 @@ ifndef HGPATH
 endif
 	echo "Installing vendor directory"
 	if [ "$(GLIDE_HOME)" ]; then mkdir -p "$(GLIDE_HOME)"; fi
-	$(GLIDE) install -v
+	timeout 20m $(GLIDE) install -v || $(GLIDE) install -v
```

Comment:
Use timeout -v and put the timeout on both glide invocations please.

Reply:
I removed -v intentionally as I see that in centos timeout does not have that option

![Screenshot from 2021-01-21 19-57-08](https://user-images.githubusercontent.com/22993933/105364306-06a4b380-5c23-11eb-8a5f-8e417b6159da.png)

Reply:
OK. We just want the 2nd invocation to also time out in case it gets stuck